src/WebSocketServer.php

```diff
@@ -211,7 +211,7 @@ private function messagesWorker(array $readSocks)
                 continue;
             }
 
-            if (method_exists($this->handler, self::MAP_EVENT_TYPE_TO_METHODS[$dataType])) {
+            if ($dataType && method_exists($this->handler, self::MAP_EVENT_TYPE_TO_METHODS[$dataType])) {
```

Comment:
Hi, thanks a lot - good catch, could u please just write this as `empty($dataType) === false && ...`, just for more readability and awareness for others?

Reply:
Solving conflicts would be great also, probably with upstream update.